plugins/modules/aci_maintenance_policy.py

```diff
@@ -21,12 +22,12 @@
     - The name for the maintenance policy.
     type: str
     aliases: [ maintenance_policy ]
-  runmode:
+  run_mode:
     description:
     - Whether the system pauses on error or just continues through it.
+    - The APIC defaults to C(pauseOnlyOnFailures) when unset during creation.
     type: str
-    choices: [ pauseOnlyOnFailures, pauseNever ]
-    default: pauseOnlyOnFailures
+    choices: [ pause_always_between_sets, pause_only_on_failures, pause_never ]
```

Comment:
missing alias "runmode"
missing previously existing choices as this will be a breaking change if we do not still accept those.